cardano-testnet/src/Testnet/Process/Cli.hs

```diff
@@ -116,6 +121,20 @@ cliNodeKeyGen tmpDir vkey skey counter = do
     ]
   return (File vkPath, File skPath, File counterPath)
 
+-- | Call a command of the CLI that returns JSON to stdout. Then parse it,
+-- and deserialize it to a Haskell value. Fail the test if a step fails.
+-- If your CLI command doesn't support
+-- returning JSON to stdout, and needs going through a file instead, probably
+-- you should add a similar function to this one.
+execCliStdoutToJson :: ()
```

Comment:
I think we should move this function to `cardano-cli` test utilities.

Reply:
@carbolymer> you mean to this file? https://github.com/IntersectMBO/cardano-cli/blob/66216d002b0171ae0e86dce0b4d4885c52d0a2aa/cardano-cli/test/cardano-cli-test-lib/Test/Cardano/CLI/Util.hs

Reply:
Maybe let's leave it here for now. Let's postpone moving this until we will need this in CLI.

Reply:
Agreed :+1: